### PR TITLE
Fix low contrast guest anchors & sponsor links

### LINKED ISSUE #118: Improve the color contrast on the home page links
The color contrast here is really bad:

screen shot 2016-06-15 at 2 54 47 pm

The colors come from [this code](https://github.com/javascriptair/site/blob/601ea1e7276af2a02f67db5d5bb3e0d92382fb1a/src/pages/home/sections/episodes/episode-small.js#L77)

Thanks!

### PR DESCRIPTION
**Pull request by rawrmonstar**
Closes #118.

### CODE REVIEW

File: src/pages/home/sections/episodes/episode-small.js [hunk 1]
```diff
@@ -83,9 +83,9 @@ EpisodeSmall.styles = StyleSheet.create({
     },
   },
   guestsAnchor: {
-    color: 'gray',
+    color: '#634806',
```
Comment (kentcdodds) [inline src/pages/home/sections/episodes/episode-small.js]: Someone fixed this one, but the other two changes are needed! Could you update your pull request to include only those changes? Thanks!

**Comment by rawrmonstar** [comment]
I think I messed up my rebase when I squashed it. The diff is still showing the `episode-small.js` file. Oops.

**Comment by kentcdodds** [comment]
I can still get this merged and fix it up if you'd like. Or you can make another PR. Whatever you wanna do :-)

**Comment by rawrmonstar** [comment]
If you can fix and merge this one then that'll be ok with me.

**Comment by kentcdodds** [comment]
Sure thing!

**Comment by kentcdodds** [comment]
Merged! It's building now. You'll be on the site as soon as it's finished building and deploying :-)

screen shot 2016-06-15 at 9 58 05 pm

**Comment by kentcdodds** [comment]
Thanks again :D

**Pull request closed without merging**